### PR TITLE
Changelog links to BitBucket backup

### PR DESCRIPTION
**Pull request by chapulina**
Pull requests and commits point to the backup.

Issues point to this repo.

### CODE REVIEW

**Comment by scpeters** [comment]
> Issues point to this repo.

it looks like the issues are pointing to the gh-pages repo

**Comment by scpeters** [comment]
I guess we'll repeat this for the changelog on other release branches too?

File: Changelog.md [hunk 1]
```diff
@@ -740,337 +740,337 @@
...
 1. Add lighting element to visual
-     * [Pull request 79](https://bitbucket.org/osrf/sdformat/pull-request/79)
+     * [Pull request 79](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/79)
 1. SDF 1.5: add flag to fix joint axis frame #43 (gazebo issue 494)
-     * [Pull request 83](https://bitbucket.org/osrf/sdformat/pull-request/83)
-     * [Issue 43](https://bitbucket.org/osrf/sdformat/issues/43)
-     * [Gazebo issue 494](https://bitbucket.org/osrf/gazebo/issues/494)
+     * [Pull request 83](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-request/83)
+     * [Issue 43](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/issues/43)
+     * [Gazebo issue 494](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/gazebo/issues/494)
```
Comment (scpeters) [inline Changelog.md:987]: gazebo issue has wrong link

Comment (scpeters) [reply Changelog.md]: 953816f

**Comment by scpeters** [comment]
> > Issues point to this repo.

> it looks like the issues are pointing to the gh-pages repo


953816f

**Comment by chapulina** [comment]
> it looks like the issues are pointing to the gh-pages repo

Oh thanks, I thought I had caught all of them.

> I guess we'll repeat this for the changelog on other release branches too?

Yeah that makes sense. I think it's worth it making a little script for this. I can do that now that the original idea has been approved.

**Comment by chapulina** [comment]
> it looks like the issues are pointing to the gh-pages repo

Ouch, I just found those changes on my workspace uncommitted :grimacing: I had also changed `[Pull request #]` to `[BitBucket pull request #]`, because the numbers may eventually overlap.

**Pull request merged**